compiler/natives/src/crypto/subtle/xor.go

```diff
@@ -0,0 +1,48 @@
+//go:build js
+// +build js
+
+package subtle
+
+func XORBytes(dst, x, y []byte) int {
+	n := len(x)
+	if len(y) < n {
+		n = len(y)
+	}
+	if n == 0 {
+		return 0
+	}
+	if n > len(dst) {
+		panic("subtle.XORBytes: dst too short")
+	}
+
+	// xorBytes(&dst[0], &x[0], &y[0], n) // arch-specific
+	// The above uses unsafe and generics for specific architecture
+	// to pack registers full instead of doing one byte at a time.
+	// We can't do the unsafe conversions from []byte to []uintptr
+	// so we'll simply do it one byte at a time.
+
+	x = x[:len(dst)] // remove bounds check in loop
```

Comment:
nit: I assume this comment got copied over from the original source. However, gopherjs compiler is not sophisticated enough to eliminate the check, so the comment is a bit misleading in this context.

Reply:
Oh, yep, that's a copy/paste. I'll remove it

Reply:
Removed